health/health_test.go

```diff
@@ -233,7 +233,7 @@ func initSpark(t *testing.T, executorType pc.SparkExecutorType, storeType fs.Fil
 		}
 	case fs.GCS:
 		credsFile := os.Getenv("GCP_CREDENTIALS_FILE")
-		content, err := ioutil.ReadFile(credsFile)
+		content, err := os.ReadFile(credsFile)
```

Comment:
Any particular reason for this?

Reply:
My Golang linter pointed out that `ioutil.ReadFile` was deprecated in 1.16 and now just calls `os.ReadFile`, so I figured I'd remove instances as I encountered them.